rootfs/etc/nginx/lua/configuration.lua

```diff
@@ -185,11 +184,126 @@ local function handle_certs()
   end
 end
 
+local function get_backends_with_name(name)
```

Comment:
I prefer to rename it to `get_backends_by_name`

No tweaking needed at the moment, see what others think

Reply:
I'm also confused about prepositions, but the same style is important. I accept your suggestion.